creator-node/src/services/sync/processSync.js

```diff
@@ -38,7 +38,8 @@ async function processSync (serviceRegistry, walletPublicKeys, creatorNodeEndpoi
     redisKey = redis.getNodeSyncRedisKey(wallet)
     let lockHeld = await redisLock.getLock(redisKey)
     if (lockHeld) {
-      throw new Error(`Cannot change state of wallet ${wallet}. Node sync currently in progress.`)
+      const errorObj = `Cannot change state of wallet ${wallet}. Node sync currently in progress.`
```

Comment:
this change is bc this code is not inside the try-catch and was incorrectly forcing this function to throw an error instead of setting the errorObj as it is designed to be done